### PR TITLE
feat: expose `*ref` props to access underlying elements

### PR DESCRIPTION
**Pull request by huntabyte**

### CODE REVIEW

File: packages/barqode/src/lib/components/types.ts [hunk 1]
```diff
@@ -119,7 +134,36 @@ export type StreamProps = {
...
+	 *
+	 */
+	videoRef?: HTMLVideoElement | null;
+
+	/**
+	 * A reference to the wrapping `div` element.
+	 *
+	 * @bindable
+	 */
+	wrapperRef?: HTMLElement | null;
```
Comment (ollema) [inline packages/barqode/src/lib/components/types.ts:154]: this one is not exposed yet right?

File: packages/barqode/src/lib/components/barqode-dropzone.svelte [hunk 2]
```diff
@@ -8,11 +8,11 @@
 		onDragover,
 		onError,
 		children,
+		inputRef = $bindable(null),
+		wrapperRef = $bindable(null),
```
Comment (huntabyte) [inline packages/barqode/src/lib/components/barqode-dropzone.svelte:12]: Exposed here @ollema

Comment (ollema) [reply packages/barqode/src/lib/components/barqode-dropzone.svelte:12]: no I meant the other component. you have defined it in both types I think

Comment (huntabyte) [reply packages/barqode/src/lib/components/barqode-dropzone.svelte:12]: Oh yeah! Done ✅

**Review by ollema — APPROVED** [review]

**Pull request merged**